src/graphql/mutations/project.ts

```diff
@@ -95,30 +95,32 @@ export const MUTATION_DELETE_PROJECT_REWARD = gql`
 `
 
 export const MUTATION_CREATE_PROJECT_MILESTONE = gql`
-  mutation CreateProjectMilestone($input: CreateProjectMilestoneInput) {
-    createProjectMilestone(input: $input) {
+  mutation ProjectGoalCreate($input: ProjectGoalCreateInput!) {
```

Comment:
might be a good idea to update the mutation name to: MUTATION_CREATE_PROJECT_GOAL